syntax = "proto3";

package primary_backup;
service PrimaryBackupgRPC {
    rpc SyncData (stream SyncDataReq) returns (SyncDataReply) {}
}
  
// Write sync data
message SyncDataReq {
    int32  offset = 1;
    bytes  buf = 2;
}

message SyncDataReply {
    int32 err = 1;
}
